content/diversity-accessibility.html

```diff
@@ -20,6 +20,8 @@
 <h2>Django Girls</h2>
 <p>There'll be a day-long <a href="https://djangogirls.org/pyconuk2016/">Django Girls</a> workshop at PyCon UK. Since the first Django Girls event in July 2014, thousands of women have been introduced to Django and Python through these free workshops.</p>
 
+<p>Free conference tickets for PyCon UK will be given to DjangoGirls attendees. These are full tickets, and include meals, refreshments and everything else that the conference has to offer.</p>
```

Comment:
Can you change "meals" to "lunches"?  Don't want people to get the wrong idea about evening meals.